ia/src/message.py

```
@@ -0,0 +1,50 @@
+import sys
+from typing import List, Tuple
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)